types/office-js-preview/index.d.ts

```diff
@@ -19919,6 +19919,455 @@ declare namespace Excel {
+        */
+        map = "Map",
+        /**
+        * Map pin icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        mapPin = "MapPin",
+        /**
+        * Microscope icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        microscope = "Microscope",
+        /**
+        * Money icon. Displays as paper money and coins.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        money = "Money",
+        /**
+        * Mountain icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        mountain = "Mountain",
+        /**
+        * Movie camera icon. Displays as a video camera.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        movieCamera = "MovieCamera",
+        /**
+        * Music note icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        musicNote = "MusicNote",
+        /**
+        * N by one icon. Displays as a three by one grid.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        nByOne = "NByOne",
+        /**
+        * N by N icon. Displays as a three by three grid.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        nByN = "NByN",
+        /**
+        * Notable people icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        notablePeople = "NotablePeople",
+        /**
+        * Notebook icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        notebook = "Notebook",
+        /**
+        * One by one icon. Displays as a one by one grid.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        oneByOne = "OneByOne",
+        /**
+        * One by N icon. Displays as a one by three grid.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        oneByN = "OneByN",
+        /**
+        * Partly sunny weather icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        partlySunnyWeather = "PartlySunnyWeather",
+        /**
+        * Person icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        person = "Person",
+        /**
+        * Pilates icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        pilates = "Pilates",
```

Comment:
I was trying to decide which one was the most oddly specific, and it's totally Pilates.